src/python/grpcio/grpc/_channel.py

```diff
@@ -1360,7 +1506,8 @@ def _deliver(state, initial_connectivity, initial_callbacks):
                 return
 
 
-def _spawn_delivery(state, callbacks):
+def _spawn_delivery(state: _ChannelConnectivityState,
+                    callbacks: Sequence[Callable]) -> None:
```

Comment:
`Sequence[Callable[[grpc.ChannelConnectivity], None]]`

Reply:
Changed to List[Callable[[grpc.ChannelConnectivity], None]]